README.md

```diff
@@ -0,0 +1,98 @@
+
+There are two other files that we've provided that you won't need to change `src/main.js` and `src/result.js`.  They are worth taking a quick look at but don't worry too much if you don't understand them fully.
+* `src/main.js` is the actual driver code for the CLI app.  If you want to manually test your code you can use this.
+* `src/result.js` is the library we've written to handle results and errors for this app.  It provides three functions, two of which you will use and one of which you're going to see in the tests.  **Every helper function should make sure to call `setResult` or `setError` before each `return`.**
+  * `setResult` call this function to set the result of your function.  We use this because Axios is asynchronous.
+  * `setError` call this function to indicate an error in your function.  Again, we use this because Axios is asynchronous.
+  *  If it takes too long for your code to produce a result (like say because you haven't written it yet) it will time out.
+
+There are tests for each of the waves in the `test/` directory, there is one file per wave.  For example you can run the Wave 1 tests with `jest test/wave1.test.js`.
+
+**Note: Because Axios is asynchronous we will need to use `setResult` instead of returning results and `setError` instead of throwing errors.**
+
+## Wave 0: Familiarize Yourself with the Pets API
+
+Read through the [documentation for the Pets API](https://github.com/AdaGold/pets-api) and experiment with using it via Postman so you can have a handle on how it works for the tasks below.
+
+## Wave 1: List Pets
+
+Before we start doing things like looking at individual pets we want to be able to get a list of all of the pets.
+
+To do this fill out the `listPets` function.  This will need to make a call to the Pets API and should `setResult` a list of objects containing (at least) the pet's `id` and `name` and should call `setError` with an error message if the request fails.
+
+You can run the Wave 1 tests with `jest test/wave1.test.js`.  Once these are passing move on to Wave 2.
+
+These tests use a mocking library to do what we used VCR to do in Ruby.  In each test you'll see a `mock.onGet` or `mock.onPost` to set up what should happen for that particular test before the actual assertions.
+
+### Function to complete
+
+* `listPets`
+
+## Wave 2: Details
+
+We now want to be able to look at individual pets.  Like in Slack CLI we're going to want to select a given pet before we ask for details on one.  To do this we're going to fill out a few functions that we're creating within a closure.  There is a provided function `selectPet` that will prompt you for a pet's `id` and save it.
+
+To do this you will need to fill out the `petDetails` function.  This will need to make a call to the Pets API and should `setResult` and object with details for the pet and should call `setError` with an error message if the request fails.
```

Comment:
Becca and I were talking, and Becca in Wave 1, Becca was a little confused about what we should put in `setResult`. Could we put some polish by making what we put into `setResult` and `setError` abundantly clear per wave within a bulleted list/aka break the requirements of each wave into bulleted lists? (possibly use Adagrams README as a model, but idk lol)